packages/Testimonial/Testimonial.jsx

```diff
@@ -37,19 +33,16 @@ const Testimonial = ({ testimonial, imgSrc, imgAlt, title, additionalInfo, ...re
 Testimonial.propTypes = {
   /** Testimonial main text */
   testimonial: PropTypes.string.isRequired,
-  /** Image URL - suggested size is 60x60px */
-  imgSrc: PropTypes.string,
+  /** Image Component - suggested & max size is 60x60px. */
+  image: componentWithName('Image'),
```

Comment:
Unfortunately, the airbnb proptype `componentWithName` bloats the final build of this component. We can either figure out how to properly tree shake this method, or use a simplified, custom validator:

```js
/* eslint-disable import/prefer-default-export */

import { warn } from './warn'

export const componentWithName = (componentName, props, propName) => {
  if (props[propName].type.name !== componentName) {
    warn(
      '@tds/community-testimonial',
      "Component supplied to the 'image' prop must be @tds/core-image"
    )
  }
}

```